tests/subsys/lw_sched/prj.conf

```diff
@@ -0,0 +1,5 @@
+CONFIG_ZTEST=y
+CONFIG_ZTEST_NEW_API=y
```

Comment:
No longer needed (if rebased on main)

Reply:
Rebasing and removing CONFIG_ZTEST_NEW_API